docs/get-started/introduction.md

```diff
@@ -106,6 +106,10 @@ only created on nodes running nfd-master.
 
 NFD takes use of some Kubernetes Custom Resources.
 
+[NodeFeature](../usage/custom-resources/nodefeature)s (EXPERIMENTAL)
```

Comment:
```suggestion

[NodeFeature](../usage/custom-resources#nodefeature)s (EXPERIMENTAL)
```

Reply:
would you mind also to fix the same issue with NodeFeatureRule and NodeResourceTopology below?

Reply:
Woops, thanks! My weary old eyes didn't spot this. Fixed